### DISCUSSION TITLE
[Solved] Mobile emulator mode cannot scroll down in Chrome Dev Tools

### DISCUSSION THREAD

**Discussion by christoferd** [thedevdojo/wave#164] [General]
Issue/bug: While testing in Mobile view (using developer tools) I'm unable to scroll down.
Title: Mobile emulator mode cannot scroll down in Chrome Dev Tools

**Reproduce:**
- Fresh install, Windows Herd auto installer
- Visit dashboard page in browser
- Press F12 (open developer tools panel)
- Switch to mobile/responsive view and select a mobile from the preset list
- Try to scroll down, notice that there is an issue here

[Solved partially] Issue is somewhat caused by zoom in dev tools mobile emulator, see response below with screenshots

**Comment by bobbyiliev** [comment]
Hi,

Interesting, seems to be working for me:

![wave-scroll](https://github.com/user-attachments/assets/f23bad45-1030-4b68-baf3-d35fe773795f)

Which theme are you using? Do you see any errors in the logs? Any other additional info might be helpful.

**Comment by christoferd** [comment]
[Solved partially] Issue is somewhat caused by zoom in dev tools mobile emulator, see response below with screenshots

PS: I don't know if this has to do with the meta "viewport" settings, or a CSS setting, or an issue with Google Dev Tools

## 75%
Setting 75% zoom in Dev Tools the form fits into the boundaries, so there is no need to scroll
![wave-3-mobile-view-dev-tools (1)](https://github.com/user-attachments/assets/14116f74-541b-4027-b305-c7e8dd64afb0)

## 100%
Setting 100% zoom in Dev Tools the shell of the phone does not fit into my browser space, the form _does not_ fit into the boundaries, there's no scroll bar outside the simulator.
Note that the viewport is scrollable, if I try to scroll a scrollbar appears, just that it's the full height.
![wave-3-mobile-view-dev-tools (3)](https://github.com/user-attachments/assets/be67d361-b728-4068-bb0e-6632b9903d57)

## 150%
Same issue
![wave-3-mobile-view-dev-tools (2)](https://github.com/user-attachments/assets/4eab222f-009a-433a-997a-c0d4bad0468b)

## Video
![5i3pnYKV2p](https://github.com/user-attachments/assets/030797ba-1581-484d-9ff6-69d9804ceb86)

**Reply by bobbyiliev** [reply]
Ok thanks for the extra details, this isn't actually a bug with Wave in any way but rather standard behavior for any website that fits entirely within the screen. When the content of a page doesn't exceed the viewport size, there's simply nothing to scroll, so the scrollbar doesn't appear.

To demonstrate, here's an example with Twitter showing the exact same behavior:  
![wave-twitter-same](https://github.com/user-attachments/assets/758c1034-44ba-45f0-a6ab-ba85b4fc8d9f)

This behavior occurs because the page content fits perfectly on the screen without requiring scrolling.
